src/popup/router/pages/PermissionsDetails.vue

```diff
@@ -124,6 +124,7 @@ export default {
   font-size: 15px;
   text-align: left;
   color: variables.$color-white;
+  padding: 6px;
```

Comment:
`padding: 16px;`
the common padding in settings screens are 16px

Reply:
https://github.com/aeternity/superhero-wallet/blob/c6ebb4fcfe282e6ccccee15de9bf01860d21fa47/src/popup/pages/PermissionsSettings.vue#L50

Reply:
I suggest to use `--screen-padding-x` for x-axis paddings to maintain consistency within the pages.

Reply:
Fixed